### ISSUE TITLE
Update: [XML External Entity Prevention Cheat Sheet]

### ISSUE THREAD

**Issue by wittjoe1** [OWASP/CheatSheetSeries#1354]
## What is missing or needs to be updated?

The chapter on .net refers to version 4.5 - nobody should really be using that anymore. It contains references to dotnet_security_unit_testing - this project was created over 7 years ago. Is this still up to date or would it not be better to exclude it in order to avoid a false sense of security?

The chapter on iOS is "up to date" with iOS 6 from 2012

## How should this be resolved?

Can you please check whether some of the content is now obsolete?
As I am neither a specialist for .net nor for iOS, this should not be my cup of tea...

**Comment by jmanico** [comment]
We accept PR's for sure. Thank you for pointing this out. Can we just remove that old content for now? What do you suggest?

**Comment by wittjoe1** [comment]
Unfortunatly i really don't know - im just translating this Cheat Sheet for my company using DeepL and Brain.exe when i recognized, that these passages are likely out of date (just my 2 cents)...

**Comment by Prasad-JB** [comment]
Good catch! I agree that references to .NET 4.5 and iOS 6 are very outdated, and keeping them might create a false sense of security for readers.

A couple of possible approaches:

.NET section - update examples to modern .NET (≥ .NET 6/7) and remove links to unmaintained projects like dotnet_security_unit_testing.

iOS section - either refresh with current APIs (iOS 17 / recent Swift guidelines) or drop the outdated content until a contributor with iOS expertise can provide an up-to-date version.

Add a general note at the top of the CS clarifying that developers should always check framework/library documentation for the latest secure defaults.

I’d be happy to help draft the .NET update if the team agrees on this direction.

**Comment by mackowski** [comment]
I think this is the good direction @Prasad-JB

**Comment by sujanyd** [comment]
@wittjoe1.   please assign this issues to me

**Comment by wittjoe1** [comment]
Unfortunatly i don't have any permessions for assigning this thread to others...
@mackowski : could you assign this to @sujanyd , please?
